js/acquia_lift_target.js

```diff
@@ -226,6 +226,11 @@ Drupal.acquiaLiftLearn = (function() {
     initializeSession();
     settings = Drupal.settings.acquia_lift_learn;
     api = Drupal.acquiaLiftV2API.getInstance();
+    for (var agent_name in Drupal.settings.acquia_lift_target.agent_map) {
+      if (Drupal.settings.acquia_lift_target.agent_map.hasOwnProperty(agent_name)) {
+        site_name_prefixes[agent_name] = Drupal.settings.acquia_lift_target.agent_map[agent_name].site_name_prefix;
```

Comment:
Right, I see what you are doing there now. This allows the old campaigns to keep running. Also, comparing to a boolean "use_site_name_prefix", storing the actual prefix allows the campaigns to keep running even when the customer_site value is changed in the future. Good call.